config/config.go

```diff
@@ -698,7 +698,7 @@ func InitializeConfig(confPath string, configCheck, configStrict bool, reloadFun
 		}
 	}
 	enforceCmdArgs(cfg)
-	globalConfHandler, err = NewConfHandler(cfg, reloadFunc, nil)
+	globalConfHandler, err = NewConfHandler(confPath, cfg, reloadFunc, nil)
```

Comment:
The cmdArgs will be stored to the config file? The behavior is reasonable？

Reply:
All `cmdArgs` are not stored now, and this may cause a problem when it restarts, for example, some config items from `PD` may be rewritten by `cmdArgs` if it restarts with the same `cmdArgs`.
This problem is too tricky and complex, so I prefer to ignore these problems now.